docs/versioned_docs/version-2.11.0/stateful-applications/cockroachdb.md

```diff
@@ -0,0 +1,294 @@
+### Install OpenEBS
+
+If OpenEBS is not installed in your K8s cluster, this can be done from [here](/docs/user_guides/installation). If OpenEBS is already installed, go to the next step.
+### Select OpenEBS storage engine
+
+A storage engine is the data plane component of the IO path of a Persistent Volume. In CAS architecture, users can choose different data planes for different application workloads based on a configuration policy. OpenEBS provides different types of storage engines and you should choose the right engine that suits your application requirements and storage available on your Kubernetes nodes. For more information you can read [here](/docs/overview#types-of-openebs-storage-engines).
+
+After OpenEBS installation, choose the OpenEBS storage engine as per your requirement. 
+
+- Choose **cStor**, If you are looking for replicated storage features and other enterprise graded features such as volume expansion, backup and restore, etc.
+
+- Choose **OpenEBS Local PV**, If you are looking for direct-attached storage or low latency data write or if the application manages data replication.
+
+In this document, we are deploying CockroachDB using OpenEBS Local PV. 
+### Configure OpenEBS Local PV StorageClass
+
+Depending on the type of storage attached to your Kubernetes worker nodes, you can select from different flavors of Dynamic Local PV - Hostpath, Device, LVM, ZFS, or Rawfile. For more information, you can read [here](/docs/concepts/localpv).
+
+The Storage Class `openebs-device` will be used to deploy CockroachDB in the Kubernetes cluster.
+
+**Note:** Ensure that you have at least one disk with the required capacity added to the corresponding nodes before CockroachDB installation. In this example, we have added one 100GB disk to each node.
+### Install CockroachDB Operator
+
+- Install the CRD using the following command.
+
+  ```bash
+  kubectl apply -f https://raw.githubusercontent.com/cockroachdb/cockroach-operator/master/config/crd/bases/crdb.cockroachlabs.com_crdbclusters.yaml
+  ```
+
+  ```bash
+  #sample output
+  customresourcedefinition.apiextensions.k8s.io/crdbclusters.crdb.cockroachlabs.com created
+  ```
+ 
+  **Note:** On GKE, additional RBAC policy of cluster-admin is required. Use the following command to create it.
+
+  ```bash
+  kubectl create clusterrolebinding $USER-cluster-admin-binding \
+  --clusterrole=cluster-admin \
+  --user=<google-cloud-user-account>
+  ```
+
+  You can obtain the Google cloud user account by using the following command 
+
+  ```bash
+  gcloud info | grep Account
+  Account: [username@mayadata.io]
+  ```
+
+  ```bash
+  #sample output
+  clusterrolebinding.rbac.authorization.k8s.io/k8s-cluster-admin-binding created
+  ``` 
+
+- Deploy the cockroachdb operator using the following command
+
+  ```bash
+  kubectl apply -f https://raw.githubusercontent.com/cockroachdb/cockroach-operator/master/manifests/operator.yaml
+  ```
+
+  ```bash
```

Comment:
```suggestion
  ```bash hideCopy
```

Reply:
Why hideCopy? because we are not allowing users to copy the result. Only `commands` and `yaml` definitions etc are allowed to copy.

---
I am not commenting for every output, please add `hideCopy` to every output.

Reply:
ok